src/main/java/org/spongepowered/api/item/inventory/type/Interactable.java

```diff
@@ -24,23 +24,20 @@
  */
 package org.spongepowered.api.item.inventory.type;
 
-import org.spongepowered.api.entity.Entity;
+import org.spongepowered.api.entity.living.player.Player;
 import org.spongepowered.api.item.inventory.Inventory;
 
 /**
- * Interface for inventories which may be interacted with by specific types of
- * Entity.
- *
- * @param <T> Base type of entity which may interact with this object  
+ * Interface for inventories which may be interacted with by Players.
  */
-public interface Interactable<T extends Entity> extends Inventory {
```

Comment:
Your point might be right, but at least remove the change on this interface. We might need further discussion when Mumfrey comments.